backend/conf/routes

```diff
@@ -12,7 +12,7 @@ POST          /api/collections/:collection/:ingestion/verifyFiles           cont
 
 GET           /api/blobs                                                    controllers.api.Blobs.getBlobs
 POST          /api/blobs/:id/reprocess                                      controllers.api.Blobs.reprocess(id, rerunSuccessful: Option[Boolean], rerunFailed: Option[Boolean])
-DELETE        /api/blobs/:id                                                controllers.api.Blobs.delete(id)
+DELETE        /api/blobs/:id                                                controllers.api.Blobs.delete(id, deleteFolders: Boolean, checkChildren: Boolean)
```

Comment:
It's not an issue with this PR since it was in the CLI before but I think we ought to think of a way of partitioning off CLI-only endpoints from the normal REST API.

Reply:
ok, sounds sensible for the future